### PR TITLE
Add HNT to fee sol

### PR DESCRIPTION
**Pull request by ChewingGlass**

### CODE REVIEW

**Review by jthiller — DISMISSED** [review]

File: docs/helium-tokens/sol.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+The Helium Network now utilizes the Solana blockchain, which is extremely fast, scalable, and
+low-cost. Solana has a separate token, which is called SOL.
+
+1. Any time you interact with the Solana blockchain, it will charge a small fee in SOL. This
+   includes sending tokens, making payments, swapping tokens, minting NFTs, claiming rewards from
+   your Hotspots, and all on-chain Helium transactions. Solana’s average transaction fee is 0.000014
+   SOL. This is massively cheaper than the Helium L1 blockchain’s fee of $0.35 worth of DC.
+2. Solana stores information, including info about your token balances, NFTs, and anything else, on
+   accounts. Storing this information takes up valuable space, so Solana charges a fee called “rent”
```
Comment (abhay) [inline docs/helium-tokens/sol.mdx]:
```suggestion
   chain as special "accounts". Storing this information takes up valuable space, so Solana charges a fee called “rent”
```

File: docs/helium-tokens/sol.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+   accounts can return rent to your wallet. Rent for one account is ~0.002 SOL, so it’s also pretty
+   cheap, but more expensive than the transaction fee. In most cases, you get this fee back once you
+   close the corresponding account. (Note: this fee may vary and is more complicated, but has a
+   current max of ~0.002 SOL.)
+
+If you don’t have enough SOL in your wallet, you will not be able to make transactions or make new
+accounts to hold tokens. Performing more complex transactions where accounts need to be created
+may require more SOL in rent than you have. This includes reasserting your Hotspot’s location,
+locking HNT to gain veHNT, and potentially more complex DeFi actions where multiple accounts need to
+be created (even if only temporarily). If you do not have enough sol, you can use this tool to swap
```
Comment (abhay) [inline docs/helium-tokens/sol.mdx]:
```suggestion
be created (even if only temporarily). If you do not have enough SOL, you can use this tool to swap
```

**Review by jthiller — DISMISSED** [review]

File: src/theme/HntToFeeSol.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  const url = siteConfig.customFields.HNT_TO_RENT_SERVICE_URL
+  const { publicKey, wallet } = useWallet()
+  const { connection } = useConnection()
+  const { execute, error, loading } = useAsyncCallback(async () => {
+    if (wallet && connection && wallet.adapter && wallet.adapter.connected) {
+      const txRaw = (await axios.post(`${url}/hnt-to-fees`, {
+        wallet: publicKey.toBase58()
+      })).data
+      const tx = Transaction.from(Buffer.from(txRaw))
+      const signed = await wallet.adapter.signTransaction(tx)
```
Comment (Perronef5) [inline src/theme/HntToFeeSol.tsx:44]: Can you try `signAndSendTransaction` and see if that works?

Comment (ChewingGlass) [reply src/theme/HntToFeeSol.tsx:44]: It says that's not a function on adapter

**Review by jthiller — DISMISSED** [review]

**Review by jthiller — APPROVED** [review]

**Pull request merged**